lms/djangoapps/instructor_task/api.py

```diff
@@ -279,7 +279,10 @@ def submit_bulk_course_email(request, course_key, email_id):
     # We also pull out the targets argument here, so that is displayed in
     # the InstructorTask status.
     email_obj = CourseEmail.objects.get(id=email_id)
-    targets = [target.target_type for target in email_obj.targets.all()]
+    targets = [
+        target.short_display()
+        for target in email_obj.targets.all()
+    ]
```

Comment:
I actually question how necessary these display names are at all. @chris-mike, how would you feel about the `to_option` information not being visible in the email task history table? The "sent to list" is still available above, in the `Sent Email History` view, and in the email detail when a message is selected.